spec/requests/admin/dashboard_spec.rb

```diff
@@ -0,0 +1,21 @@
+describe Admin::DashboardController, type: :request do
+  let(:user) { create :user }
+  let!(:role) { create :user_role, user: user, admin: admin_flag }
+  before { login user }
+
+  describe "can login to active_admin only admin role" do
+    before { get admin_root_path }
+
+    context "admin user" do
+      let(:admin_flag) { true }
+      it { expect(response).to have_http_status :success }
+      it { expect(response.body).to match 'ダッシュボード' }
+    end
+
+    context "not admin user" do
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)